tools/docker/sles.docker

```diff
@@ -19,7 +19,16 @@ RUN zypper --gpg-auto-import-keys install -y \
     gcc-c++ \
     gdb \
     git \
-    python3-pip
+    python3-pip \
+    rpm-build
+
+#addition of repos for packages
+RUN OPENSUSE_REPO=https://download.opensuse.org/repositories && \
+    zypper addrepo ${OPENSUSE_REPO}/science/SLE_15_SP4/science.repo && \
```

Comment:
I stole the text from the DevOps team.  It does appear I really only need "devel:languages:perl.repo" 
This happens because of...
`Problem: nothing provides 'perl-File-BaseDir' needed by the to be installed hipcc-1.0.0.50600-sles154.67.x86_64` 
And that doesn't happen in rocm5.6 but does in the upcoming release.